packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -75,23 +75,24 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
   const { isTranscodingEnabled } = useFFmpeg();
   const {
     muteExistingVideo,
-    isResourceTrimming,
+    isElementTrimming,
     isNewResourceMuting,
     canTranscodeResource,
   } = useLocalMedia(
     ({
-      state: { canTranscodeResource, isNewResourceMuting, isResourceTrimming },
+      state: { canTranscodeResource, isNewResourceMuting, isElementTrimming },
       actions: { muteExistingVideo },
     }) => ({
       canTranscodeResource,
       isNewResourceMuting,
-      isResourceTrimming,
+      isElementTrimming,
       muteExistingVideo,
     })
   );
   const resource = getCommonValue(selectedElements, 'resource');
   const { isMuted, id: resourceId = 0 } = resource;
-  const isTrimming = isResourceTrimming(resourceId);
+  const elementId = getCommonValue(selectedElements, 'id');
+  const isTrimming = isElementTrimming(elementId);
```

Comment:
Multiple elements can be selected. So we should only do `isElementTrimming` if one element is selected. So something like this. 

```
const isTrimming = selectedElements.length === 1 ? isElementTrimming(elementId) : false;
```

Reply:
I don't see why this is needed 🤔 

We do `const elementId = getCommonValue(selectedElements, 'id');` above

So if there is a multi-selection, `elementId` will be `MULTIPLE_VALUE` (which is the string `((MULTIPLE))`)

`isElementTrimming('((MULTIPLE))')` does not cause any harm

Reply:
It seems a little wasteful there, there is no need to `isElementTrimming('((MULTIPLE))')`. It will do a lookup in state when it not needed. The `selectedElements.length === 1` in an inexpensive way of avoiding that.